libsignal-protocol/src/address.rs

```diff
@@ -63,3 +82,30 @@ impl<'a> Clone for Address<'a> {
         })
     }
 }
+
+impl<'a> Ord for Address<'a> {
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.device_id().cmp(&other.device_id())
```

Comment:
i'm not sure if we have to compare the `name` too ? and if we have, how?

Reply:
I think we should. I remember reading somewhere in their documentation/code that an `Address` is a tuple of the recipient name and device ID.